### ISSUE TITLE
Not compatible with Final Cut Pro 10.7.1

### ISSUE THREAD

**Issue by wih27631** [latenitefilms/TransferToolbox#2]
I'm trying to create a Library in FCPX desktop in order to use Transfer Toolbox to turn it into an iPad version. I get this error. 
<img width="601" alt="Screenshot 2024-01-13 at 5 32 36 PM" src="https://github.com/latenitefilms/TransferToolbox/assets/156556116/c3dbec77-21ae-412c-b7fb-b571a7fd6e17">
I'm using the most up-to-date version of FCPX on Sonoma, which is also up-to-date. Only one Event, named Default 01-13-24. Library was created from FCPX. Not sure what's wrong?

**Comment by latenitefilms** [comment]
Was the Library created in Final Cut Pro 10.7 or later? If not, maybe try making a new Library then dragging your Events into it?

**Comment by wih27631** [comment]
Yes. I'm on the latest version of FCPX. On Jan 13, 2024, at 6:35 PM, Chris Hocking ***@***.***> wrote:﻿
Was the Library created in Final Cut Pro 10.7 or later? If not, maybe try making a new Library then dragging your Events into it?

**Comment by latenitefilms** [comment]
I understand you're using the latest version of Final Cut Pro, but did you CREATE the library on the latest version or is it a library that's been started on an older version and updated?

**Comment by wih27631** [comment]
Yes. I originally updated a Library that was older. When it failed, I realized that probably wasn't a good plan. After that, I created new libraries in FCPX and got the same error each time. I tried with content, without, with and without a project. Always the same error. WarrenOn Jan 13, 2024, at 6:50 PM, Chris Hocking ***@***.***> wrote:﻿
I understand you're using the latest version of Final Cut Pro, but did you CREATE the library on the latest version or is it a library that's been started on an older version and updated?

**Comment by latenitefilms** [comment]
Thanks for confirming! Will investigate ASAP. Stay tuned.

**Comment by wih27631** [comment]
Cool. ThanksOn Jan 13, 2024, at 7:47 PM, Chris Hocking ***@***.***> wrote:﻿
Thanks for confirming! Will investigate ASAP. Stay tuned.

**Comment by latenitefilms** [comment]
Thanks for reporting! This should now be fixed in Transfer Toolbox v1.1.1.

**Comment by wih27631** [comment]
Hey, Chris. The updated version worked. Nice job, and thanks for such a fast response. Although all of my Libraries now convert, I had several that failed to import to my iPad FCPX. I think this was due to including too many 3rd party plugins in the test Project. When I created a new library with just one video clip, it imported properly. I then sent it back to the desktop, and added a single 3rd party title, converted it, and sent it back to the iPad. This one was imported on the iPad, and it just showed that the title couldn’t be read (red), which is easy to understand. 

Thanks for your efforts. 
 
Warren
